### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by luigi311** [pmaji/eth_python_tracker#1]
Since there are a couple of dependencies you should do a requirements file so people can install all the dependencies with pip. To create a requirements file all you need to do is setup a virtual env and only install the required python packages then do "pip freeze > requirements.txt".  With that it will output all the names of the dependencies along with the version you were using.

**Comment by arsenicks** [comment]
It's the first time I'm doing this but I think I got it correctly, here's the requirements.txt

$ pip install -r requirements.txt
Anybody can test it ?

[requirements.txt](https://github.com/pmaji/eth_python_tracker/files/1684012/requirements.txt)

**Comment by pmaji** [comment]
@luigi311 Thanks so much for opening up this discussion. I don't have any experience doing this in a virtual environment and should probably look into that, but I looked over the list that @arsenicks put together, checked it against my own view locally, and it checks out. As such, I integrated it into the file list. Great help guys; really appreciate it and let me know if you think anything was done improperly / should be re-done.
